Could not resolve dependency: peer @auth/core@"0.5.1" from @solid-auth/base@2.0.3

I am following the getting started guide https://start.solidjs.com/getting-started/project-setup and tried using the auth-js example but get the following error upon running `npm install`.

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: v10 - SolidStart@undefined
npm ERR! Found: @auth/core@0.3.0
npm ERR! node_modules/@auth/core
npm ERR!   @auth/core@"^0.3.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @auth/core@"0.5.1" from @solid-auth/base@2.0.3
npm ERR! node_modules/@solid-auth/base
npm ERR!   @solid-auth/base@"^2.0.3" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

--- a/examples/with-authjs/package.json
+++ b/examples/with-authjs/package.json
@@ -16,7 +16,7 @@
     "vite": "^4.1.4"
   },
   "dependencies": {
-    "@auth/core": "^0.3.0",
+    "@auth/core": "^0.5.1",
     "@solid-auth/base": "^2.0.3",
     "@solidjs/meta": "^0.28.2",
     "@solidjs/router": "^0.8.2",

